api/tests/test_tasks.py

```diff
@@ -119,11 +118,8 @@ def test_create_delivery_sends_email(self, mock_logger, mock_email):
         }
 
         message = render_to_string('contact_template.txt', context)
+        self.assertIn(f"{koiki.wcfmmp_host}area-privada/orders-details/33", message)
```

Comment:
wait, does that endpoint exist already? It doesn't right? we should remove it from the message if so.